Rakefile

```diff
@@ -24,6 +26,8 @@ CLOBBER.append(BENCHMARK_UPSTREAM_PB)
 CLOBBER.append(BENCHMARK_PROTOBOEUF_PB)
 CLOBBER.append(WELL_KNOWN_PB)
 
+protoc_import_path_string = "'#{well_known_types.pathmap("%d").uniq.join("':'")}'"
```

Comment:
we should pass an array as the command and then we won't have to do all the weird quoting here.

Reply:
Done! https://github.com/Shopify/protoboeuf/pull/168/commits/53f28ca99e8be24e2d436531fd2e8a7049190574